app/controllers/identities/registrations_controller.rb

```diff
@@ -69,7 +69,7 @@ def sign_up_params
 
   def identity_params
     params[:identity][:phone]                         = sanitize_phone(params[:identity][:phone])
-    params[:identity][:professional_organization_id]  = params[:project_role][:identity_attributes][:professional_organization_id]
+    params[:identity][:professional_organization_id]  = params[:project_role].nil? ? nil: params[:project_role].nil? ? nil:  params[:project_role][:identity_attributes][:professional_organization_id]
```

Comment:
@ying-pbrc Is there a typo on this line? `params[:project_role].nil? ? nil:` is duplicated

Reply:
Oops, you are absolutely correct, I copied and pasted little too much!